src/containers/shared/components/Transaction/AccountSet/Description.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import { ACCOUNT_FLAGS } from '../../../transactionUtils'
+import DomainLink from '../../DomainLink'
+import { TransactionDescriptionProps } from '../types'
+import { AccountSet } from './types'
+import { Account } from '../../Account'
+
+export const Description = ({
```

Comment:
Is this even needed? Seems like this is the same as what's in `Simple`

Reply:
The types of values shown are the same but it prints them out into sentences.  For example NFTokenMinter it shows

> It sets rXMART8usFd5kABXCayoP6ZfB35b4v43t as the authorized minter for this account